app.js

```diff
@@ -1,12 +1,25 @@
 const express = require("express");
 const { getCategories } = require("./controllers/categories.controllers");
+const { getReviewById } = require("./controllers/reviews.controllers");
 const app = express();
+app.use(express.json());
 
 app.get("/api/categories", getCategories);
+app.get("/api/reviews/:review_id", getReviewById);
+
+// Error Handlers
 
-//Error Handlers
 app.use((err, req, res, next) => {
   if (err.status && err.msg) {
+    res.status(err.status).send({ msg: err.msg });
+  } else {
+    next(err);
+  }
+});
+
+app.use((err, req, res, next) => {
+  let psqlErrors = "22P02";
+  if (psqlErrors.includes(err.code)) {
```

Comment:
I would probably just change this to either have psqlErrors be an array of possible bad request errors, or if that psqlErrors is a string then `psqlErrors === err.code`